### DISCUSSION TITLE
Change Camera3D gizmo shape according to viewport size

### DISCUSSION THREAD

**Discussion by timothyqiu** [godotengine/godot-proposals#5781] [3D]
It's currently always a square.

The aspect ratio could match the viewport aspect ratio.

**Comment by Calinou** [comment]
Feel free to open a pull request for this :slightly_smiling_face: 

Note that whenever the viewport width/height project setting is changed, gizmos need to be updated. The project's width/height aspect ratio should be used, *not* the editor window's aspect ratio.

**Comment by akien-mga** [comment]
Implemented by https://github.com/godotengine/godot/pull/68682.
